app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect('cinema.db3')
+        self.table_name = 'actors'
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(
+            f'INSERT INTO {self.table_name} (first_name, last_name) VALUES (?)',
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self):
+        actors_cursor = self._connection.execute(
+            f'SELECT id, first_name, last_name FROM {self.table_name}'
```

Comment:
Use select * 
Because when you change your actor class you also must to edit code here